libs/minisat/System.cc

```diff
@@ -73,7 +73,7 @@ static inline int memReadPeak(void)
 }
 
 double Minisat::memUsed() { return (double)memReadStat(0) * (double)getpagesize() / (1024*1024); }
-double Minisat::memUsedPeak(bool strictlyPeak) { 
+double Minisat::memUsedPeak(bool strictlyPeak) {
```

Comment:
Changes to `libs/minisat/` should be reflected in the patch files in that directory. For example, this could go in `libs/minisat/00_PATCH_typofixes.patch`. The other stuff maybe in a new file such as `libs/minisat/00_PATCH_wasm.patch`.